### ISSUE TITLE
restart and swarm plot figures

### ISSUE THREAD

**Issue by arijitlaik** [rbeucher/UWGeodynamics#56]
after restarting a model, the swarm plots( defined in _plots.py ) don't work, as in they plot the initial model state,  mesh plots do not show this behaviour, affected ones are the material and viscosity fields. possible add a proper manual for restart???

**Comment by rbeucher** [comment]
https://github.com/rbeucher/UWGeodynamics/wiki/Restart

Not perfect but that's a start...

**Comment by arijitlaik** [comment]
Thanks, I'll test and close this issue

**Comment by arijitlaik** [comment]
Nah, didn't work. for my use case. i'll send you an email. maybe?

**Comment by arijitlaik** [comment]
Here is the minal example to reporduce the bug
[UWGeo_postsolveFigure.zip](https://github.com/rbeucher/UWGeodynamics/files/2439758/UWGeo_postsolveFigure.zip)

**Comment by rbeucher** [comment]
Yeah I would not call that a bug. There no way I can take into account the postSolveHook automatically in a restart.

Try to call Model.restart(step, restartFolder) before defining your hook and then run
Model.run

**Comment by arijitlaik** [comment]
yup makes sense. i did remember back in kolkata i did  the workaround you suggested. I was hoping the apart from the predined plot options, user could get a choice to output figures, in their custom range colourmaps etc, set via their rcParams, moreover can you check if the outputgluciferfigures is working with model.run. haven't tried it in while.
and Do I close this issue you mark this as an enhancement

**Comment by rbeucher** [comment]
Same here. I use Paraview

**Comment by rbeucher** [comment]
Leave it for now
